src/components/entity/ha-entity-lock.html

```diff
@@ -0,0 +1,154 @@
+<link rel="import" href="../../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../../bower_components/paper-icon-button/paper-icon-button.html">
+
+<dom-module id="ha-entity-lock">
+  <template>
+    <style>
+      :host {
+        white-space: nowrap;
+      }
+
+      paper-icon-button {
+        color: var(--primary-text-color);
+        transition: color .5s;
+      }
+
+      paper-icon-button[state-active] {
+        color: var(--primary-color);
+      }
+
+      paper-toggle-button {
+        cursor: pointer;
+        --paper-toggle-button-label-spacing: 0;
+        padding: 13px 5px;
+        margin: -4px -5px;
+      }
+    </style>
+
+    <template is='dom-if' if='[[!stateObj.attributes.assumed_state]]'>
```

Comment:
In "assumed_state" mode you should still show the buttons, but without deciding which one is active

Reply:
Yes this can be a problem. I will provide improvments.